revolv/static/js/base/project.js

```diff
@@ -0,0 +1,97 @@
+$(document).ready(function () {
+	draw()
```

Comment:
Did jshint not catch this? You're missing a semicolon and this line is indented with a tab instead of a space